.github/workflows/ci.yml

```diff
@@ -110,6 +110,44 @@ jobs:
         with:
           target: ${{ matrix.target }}
 
+  test-emu:
```

Comment:
nah, don't ^C^V, don't jungle target names in CI, thus would introduce only more confusion